src/main/java/org/zaproxy/zap/extension/jwt/attacks/SignatureAttack.java

```diff
@@ -362,7 +387,8 @@ public boolean executeAttack(ServerSideAttack serverSideAttack) {
             return this.executeCustomPrivateKeySignedJWTTokenAttack()
                     || this.executeAlgoKeyConfusionAttack()
                     || this.executeNullByteAttack()
-                    || this.executePubliclyWellKnownHMacSecretAttack();
+                    || this.executePubliclyWellKnownHMacSecretAttack()
+                    || this.executeIncorrectSignatureAttack();
```

Comment:
This attack should have more priority than other attacks, so can you please add this before `executeAlgoKeyConfusionAttack` ?

Reply:
Updated